kinder/ci/workflows/skew-master-on-stable.yaml

```diff
@@ -6,7 +6,7 @@ summary: |
   config    > https://github.com/kubernetes/test-infra/blob/master/testgrid/config.yaml
 vars:
   kubeadmVersion: "{{ resolve `ci/latest` }}"
-  kubernetesVersion: "{{ resolve `release/stable` }}"
+  kubernetesVersion: "{{ resolve `release/latest-1.14` }}"
```

Comment:
i think we need to rename the files too to not have stable in the name, which means we need to update test-infra too.
should happen after 1.15 obviously.

```
C) don't use "stable" for naming.

instead of calling the current "latest upgrade job" stable->master job, call it "1.14->master".
when the new release branch is created rename this job to 1.14->1.15.
add a new job that does 1.15->master testing.
```

i think we should stop using both `stable` and `master` and use versions.
cc @fabriziopandini for confirmation.